### PR TITLE
Feature/hbase kerberos

### PR DESCRIPTION
**Pull request by vglushak**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,5 +18,5 @@ lib_managed/
 src_managed/
 project/boot/
 project/plugins/project/
-
+*.file
```
Comment (vglushak-vt) [inline .gitignore]: Not sure if we should ignore all files with .file extension.

File: api/app/controllers/v1/feedback/FeedbackController.scala [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  *
+  * @param config
+  */
+@Api("Feedback")
+class FeedbackController @Inject()(implicit val config: Config) extends Controller with FeedbackStore {
+  private[this] val logger = LoggerFactory.getLogger(getClass)
+
+
+  // public api
+  @ApiOperation(value = "Send feedback via email",
```
Comment (vglushak-vt) [inline api/app/controllers/v1/feedback/FeedbackController.scala]: Update API description!

File: api/app/controllers/v1/feedback/FeedbackController.scala [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    Json.fromJson[FeedbackObj](json) match {
+      case JsSuccess(feedbackObj, _) => {
+        logger.debug(s"Feedback Received: $feedbackObj")
+        Ok(store(feedbackObj))
+      }
+      case JsError(err) =>
+        logger.error(s"Invalid Feedback! Please give properly parsable feedback $json -> $err")
+        BadRequest(s"Invalid Feedback! Please give properly parsable feedback $json -> $err")
```
Comment (vglushak-vt) [inline api/app/controllers/v1/feedback/FeedbackController.scala]: Define variable with this text and reuse it.
```
val msg = s"..."
logger.error(msg)
BadRequest(msg)
```

**Comment by awharris** [comment]
No longer relevent

**Pull request closed without merging**